pkg/types/defaults/machinepools.go

```diff
@@ -17,4 +19,13 @@ func SetMachinePoolDefaults(p *types.MachinePool, platform string) {
 	if p.Hyperthreading == "" {
 		p.Hyperthreading = types.HyperthreadingEnabled
 	}
+	if p.OsEncryption == "" {
+		switch platform {
+		case aws.Name:
+			p.OsEncryption = types.EncryptionStrategyDisabled
```

Comment:
it's best if we moved the defauts for platforms to platform-packages.

Reply:
done, please confirm this is what you had in mind.